src/ast/passes/codegen_llvm.cpp

```diff
@@ -612,7 +612,8 @@ void CodegenLLVM::visit(Call &call)
     AllocaInst *buf = b_.CreateAllocaBPF(bpftrace_.strlen_, "path");
     b_.CREATE_MEMSET(buf, b_.getInt8(0), bpftrace_.strlen_, 1);
     call.vargs->front()->accept(*this);
-    b_.CreatePath(ctx_, buf, expr_, call.loc);
+    b_.CreatePath(
+        ctx_, buf, b_.CreateIntToPtr(expr_, b_.getInt8PtrTy()), call.loc);
```

Comment:
I'm not aware of any case where `expr_` is not either a `int8 *` (in that case nothing happens, since `CreateCast` called by `CreateIntToPtr` returns its argument unchanged if it already is of the target type) or an `uint64` (in that case the `inttoptr` is valid). I also checked that both `path(args->path)` in `vfs_truncate` and `path(args->file->path)` in `vfs_fallocate` pass LLVM IR verifier.

However, it might still be of use to create a pointer cast should `expr_` be a pointer, since the user might theoretically use of `path` on a different argument that is a pointer different from `int8 *`, hence I'll fix that.